Vector.cpp

```diff
@@ -1,8 +1,21 @@
 #include "Vector.h"
 #include <assert.h>
+#include <iostream>
+
+using namespace std;
 
 namespace Micromouse
 {
+	direction operator%(const direction &dir1, const direction &dir2)
```

Comment:
I really think it makes more sense from a clarity point of view to make it a new function rather than an overload. but I do think it would make more sense as a + overload then a % overload. as far as adding a dir - dir we really don't need to for any reason, and the more I think about it the more of a headache it sounds like to have different people using dir - dir and dir + dir in different parts of the code